dotCMS/src/main/java/com/dotmarketing/portlets/categories/business/CategoryAPIImpl.java

```diff
@@ -565,8 +566,12 @@ public boolean isParent(Category givenChild, Category givenParent, User user) {
 		return false;
 	}
 
-
-
-
+	public synchronized String suggestVelocityVarName(final String categoryVelVarName) throws DotDataException {

```

Comment:
as in my comment before, it's based on the same code we have for ContentTypeAPI that generates the VelocityVarName for Content Types. Categories had this loose end.